src/iinit.cpp

```diff
@@ -242,7 +255,7 @@ int main( int argc, char **argv )
         }
 
         if ( i != 0 ) {
-            rodsLogError( LOG_ERROR, i, "Save Password failure" );
+            rodsLogError( LOG_ERROR, i, "Failed to save password!" );
```

Comment:
same here.

Reply:
I'll change it to a `.` here since the status code gets printed on the same line right after the message. It gets a bit confusing with no punctuation in my eyes.

Original message below for example:
`ERROR: Save Password failure status = -900000 FILE_OPEN_ERR`

Reply:
period is fine fine.  thanks.